src/linux/ebpf.cpp

```diff
@@ -96,4 +98,51 @@ Try<int> load(const Program& program)
   return *fd;
 }
 
+namespace cgroups2 {
+
+Try<Nothing> attach(int fd, const string& cgroup)
+{
+  Try<int> cgroupFd = os::open(cgroup, O_DIRECTORY | O_RDONLY);
```

Comment:
we have to close this file descriptor always on exit of this function, or we will leak open file descriptors